test_files/import_equals/exporter.d.ts

```diff
@@ -0,0 +1,3 @@
+export declare namespace exportedSymbol {
```

Comment:
I found the name `exportedSymbol` confusing when reading it in a module that imports it. I think modules need to have some sort of names to differentiate them - `Top,Middle,Bottom` or `Consumer, Producer` or my favorite - `A,B,C`. Exporter is just too confusing since every module exports.

Reply:
Interesting. I find that `A,B,C` is really hard for me to understand, same for top middle bottom. Those depend on what direction you think your imports point (or in the case of A, B, C, might just be 100% random).

Most tsickle tests use `exporter` for the file that exports a symbol that's needed by the test, with the other file being the actual test. That is, `exporter` is always test setup, and is always the module whose only relevance is exporting a symbol.

We could do a PR to move to consumer / producer or similar, but I'm not 100% sold that that'd make things easier to understand.

Reply:
I see, as long as we have a good rule-of-thumb as you describe.